client/scss/components/_listing.scss

```diff
@@ -422,6 +422,7 @@ ul.listing {
 }
 
 .image-choice {
+    display: block;
```

Comment:
Without this, these are inline links even though they are displayed as blocks, and their outline looks strange.

Reply:
why not add this as an inline comment there? Will save future headaches

Reply:
If you're troubleshooting the code for this bit of the UI I think it's quite obvious that those links are not meant to be inline, so I think the commit history would be enough. But yep, the only impact in practice is the outline so might be worth adding a comment since it's not too obvious when interacting with the UI.